### ISSUE TITLE
KataGo in unreal engine ?

### ISSUE THREAD

**Issue by hydrogenpi** [lightvector/KataGo#257]
I found this Go asset here in UE marketplace:
https://www.unrealengine.com/marketplace/en-US/product/go-set

I have a sandbox game (its not related to Go at all, and is not a Go game) and just for kicks wanted to create a Go table that user can go sit down and play a game of Go for fun, like a minigame within a larger game. I found the 3D assets on the marketplace for Unreal Engine but then needed an AI engine and what better than KataGo?

Are we allowed to do this or not?

**Comment by OmnipotentEntity** [comment]
Check the licenses.

https://github.com/lightvector/KataGo/blob/master/LICENSE

References:

https://github.com/lightvector/KataGo/blob/master/cpp/external/clblast/LICENSE
https://github.com/lightvector/KataGo/blob/master/cpp/external/half-2.1.0/LICENSE.txt
https://github.com/lightvector/KataGo/blob/master/cpp/external/nlohmann_json/json.hpp
https://github.com/lightvector/KataGo/blob/master/cpp/external/tclap-1.2.1/COPYING
https://github.com/lightvector/KataGo/blob/master/cpp/external/zstr/LICENSE
https://github.com/lightvector/KataGo/blob/master/cpp/core/sha2.cpp

Nearly all of the licenses are MIT or MIT-compatible.  clblast is the exception.  It is Apache.

Long story short, yes.  Just abide by the terms of the licenses.

**Comment by lightvector** [comment]
Yep, what OmnipotentEntity said. Thanks for checking and asking. Closing this issue since it seems pretty clear.
